src/VisualStudio/Core/Test.Next/Remote/RemoteHostClientServiceFactoryTests.cs

```diff
@@ -143,18 +144,22 @@ private static async Task PerformSearchesAsync(RemoteHostClient client, Solution
             var count = 0;
             await foreach (var result in stream)
             {
+                Assert.True(text.ToString(result.DeclaredSymbolInfo.Span).StartsWith(name));
```

Comment:
this actually verifies that the positions we get back are correct in the document.